### DISCUSSION TITLE
Some intermediate samples of training vits-2 (non native EN | customer service phone calls dataset)

### DISCUSSION THREAD

**Discussion by p0p4k** [p0p4k/vits2_pytorch#18] [Show and tell]
> Here is a [samples](https://drive.google.com/drive/folders/16Pvm48wUMIHcN2IdhPIGLFi7HAycoE8H?usp=sharing) of Voice generated using a custom dataset in LJSpeech Format. 
> 
> The model ckpts used was G_70000.pth which is so early. Trained on a 2V100 Engine. 

Thanks for sharing @athenasaurav ! (#17)

**Comment by p0p4k** [comment]
You can add more samples as you progress @athenasaurav. Maybe a comparison of another model samples and vits-2 model samples would be great as well later on! Thanks.

**Comment by athenasaurav** [comment]
Sure @p0p4k

**Comment by nmfisher** [comment]
Thanks for the great repo, awesome work. Figured I should at least do you the courtesy of sharing some samples (roughly 110k iterations).

https://drive.google.com/drive/folders/13H5qeZ6U-jJVas76ln4TnukT3q1CK6eb?usp=sharing

(If the content is a bit strange, it's because it was all generated via LLaMa 2 in a little demo project I put together 
https://youtu.be/GHBOy-duE98).

Chinese will follow soon.

It's still a little unstable in parts (and it does struggle with short phrases), I'm not sure whether that's an artifact of the model, the dataset or the parameters I was training with.

**Reply by p0p4k** [reply]
Hi, what dataset did you train this on? Thanks for sharing the samples!

**Reply by nmfisher** [reply]
This is a proprietary multilingual (English/Chinese) dataset, tokenized with ARPABet/pinyin symbols (no word boundaries either, which I think may actually help).

**Reply by p0p4k** [reply]
Is there a reference sample to compare improvement from VITS-1 ; trained under same conditions.

**Reply by athenasaurav** [reply]
Yes @nmfisher the audio looks very good. Maybe you show the bad example you mentioned. Also can you give a but specifics about dataset size etc. that will be great.

**Comment by yihuitang** [comment]
Hi @nmfisher, English sample voices sound very good. Do you have any sample audio for Chinese?
